src/Filter/Filter.php

```diff
@@ -17,41 +20,11 @@
  */
 namespace DocHeader\Filter;
 
-/**
- * @author  Jefersson Nathan <malukenho@phpse.net>
- * @license MIT
- */
-final class Filter
```

Comment:
This is a BC break

Reply:
Yep. It will be a major release since I'll get rid of php >=5.5 as well